src/NLog/Targets/Target.cs

```diff
@@ -336,12 +333,13 @@ internal void Close()
                     }
                     catch (Exception exception)
                     {
+                        InternalLogger.Error(exception, "Error closing target '{0}'.", this);
+
                         if (exception.MustBeRethrown())
                         {
                             throw;
                         }
```

Comment:
This is superfluous now and can be removed, as the exception is rethrown anyway.